tests/blackbox/validation.md

```diff
@@ -0,0 +1,51 @@
+# Evaluation
+
+To find out a model that can judge the test scenarios, we need to validation the best LLM model. This can be done by running the validation tests.
```

Comment:
```suggestion
To find a model that can evaluate the test scenarios, we need to validate the best large language model (LLM) model. This can be done by running the validation tests.
```